exchanges/binanceus/binanceus_test.go

```diff
@@ -0,0 +1,1929 @@
+		t.Errorf("Binanceus CreatenewOCOOrder() error expected %v, but found %v", errIncompleteArguments, er)
+	}
+	_, er = bi.CreateNewOCOOrder(
+		context.Background(),
+		&OCOOrderInputParams{
+			Symbol:               "XTZUSD",
+			Price:                100,
+			StopPrice:            3,
+			StopLimitPrice:       2.5,
+			Side:                 order.Buy.String(),
+			Quantity:             1,
+			StopLimitTimeInForce: "GTC",
+			RecvWindow:           6000,
+		})
+	if er != nil && !strings.Contains(er.Error(), "Precision is over the maximum defined for this asset.") {
+		t.Error("Binanceus CreateNewOCOOrder() error", er)
+	}
+}
+
+var ocoOrderJSON = `{
+	"orderListId": 27,
+	"contingencyType": "OCO",
+	"listStatusType": "EXEC_STARTED",
+	"listOrderStatus": "EXECUTING",
+	"listClientOrderId": "h2USkA5YQpaXHPIrkd96xE",
+	"transactionTime": 1565245656253,
+	"symbol": "LTCBTC",
+	"orders": [
+	  {
+		"symbol": "LTCBTC",
+		"orderId": 4,
+		"clientOrderId": "qD1gy3kc3Gx0rihm9Y3xwS"
+	  },
+	  {
+		"symbol": "LTCBTC",
+		"orderId": 5,
+		"clientOrderId": "ARzZ9I00CPM8i3NhmU9Ega"
+	  }
+	]
+  }`
+
+func TestGetOCOOrder(t *testing.T) {
+	t.Parallel()
+	var resp OCOOrderResponse
+	if er := json.Unmarshal([]byte(ocoOrderJSON), &resp); er != nil {
+		t.Error("Binanceus decerializing OCOOrderResponse error", er)
+	}
+	if !areTestAPIKeysSet() {
+		t.SkipNow()
+	}
+	_, er := bi.GetOCOOrder(context.Background(), &GetOCOOrderRequestParams{})
+	if !errors.Is(er, errIncompleteArguments) {
+		t.Errorf("Binanceus GetOCOOrder() error  expecting %v, but found %v", errIncompleteArguments, er)
+	}
+	_, er = bi.GetOCOOrder(context.Background(), &GetOCOOrderRequestParams{
+		OrderListID: "123445",
+	})
+	if er != nil && !strings.Contains(er.Error(), "Order list does not exist.") {
+		t.Error("Binanceus GetOCOOrder() error", er)
+	}
+}
+
+func TestGetAllOCOOrder(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.SkipNow()
+	}
+	_, er := bi.GetAllOCOOrder(context.Background(), &OCOOrdersRequestParams{})
+	if er != nil {
+		t.Error("Binanceus GetAllOCOOrder() error", er)
+	}
+}
+
+func TestGetOpenOCOOrders(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.SkipNow()
+	}
+	_, er := bi.GetOpenOCOOrders(context.Background(), 0)
+	if er != nil {
+		t.Error("Binanceus GetOpenOCOOrders() error", er)
+	}
+}
+
+func TestCancelOCOOrder(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() || !canManipulateRealOrders {
+		t.SkipNow()
+	}
+	_, er := bi.CancelOCOOrder(context.Background(), &OCOOrdersDeleteRequestParams{})
+	if !errors.Is(er, errIncompleteArguments) {
+		t.Errorf("Binanceus CancelOCOOrder() error expected %v, but found %v", errIncompleteArguments, er)
+	}
+}
+
+// OTC end Points test code.
+func TestGetSupportedCoinPairs(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.SkipNow()
+	}
+	_, er := bi.GetSupportedCoinPairs(context.Background(), currency.Pair{Base: currency.BTC, Quote: currency.USDT})
```

Comment:
This returns `[]*CoinPairInfo` we can change that to `[]CoinPairInfo` as it's a slice anyway.  Also on trades, it's not really necessary to have a slice of pointers in this context. So if you can go over the return types and remove the pointer from the []*type. That would be better.